DOCSP-27358 pt3 Leafy Green updates (#486)
mongodb/docs-compass: source/includes/fact-favorite-query.rst

.. procedure::
   :style: normal

   .. step:: Open query history.

>       Click the :guilabel:`History` button at the top of the pipeline 
>       builder.

Reviewer: Not from these changes, but I'm thinking we can fix this up here: 
```suggestion
      Click the :guilabel:`History` button on the query bar.
```
The `query/recent/#view-recent-queries` page or the screenshot below in `query/queries/#using-the-favorites-tab` might be nice to reference here as well.

Author: Done.
